contract/src/msgs/staking/query.rs

```diff
@@ -26,6 +27,19 @@ impl QueryHandler for QueryMsg {
             }
             QueryMsg::IsExecutorEligible(query) => query.query(deps, env)?,
             QueryMsg::GetStakingConfig {} => to_json_binary(&state::STAKING_CONFIG.load(deps.storage)?)?,
+            QueryMsg::GetExecutors { offset, limit } => {
```

Comment:
This is technically new behaviour, still not sure how we want to approach this.

Reply:
Ah forgot about that...

Reply:
I think it should be fine right? It's not that crazy of a change. If not I can pull it out of this PR and put it in a separate one I suppose

Reply:
Yes, might be a good idea. Even though it is just a only-read function.

Reply:
It's been moved to #271